src/stream.rs

```diff
@@ -564,6 +570,11 @@ impl SendBuf {
         Ok(out)
     }
 
+    /// Return current max_data
+    pub fn get_max_data(&mut self) -> usize {
```

Comment:
```suggestion
    pub fn get_max_data(&self) -> usize {
```
Can remove the mutable if just reading.